pkg/monitoring/metrics/forklift-controller/migration_metrics.go

```diff
@@ -0,0 +1,110 @@
+package forklift_controller
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Calculate Migrations metrics every 10 seconds
+func RecordMigrationMetrics(c client.Client) {
```

Comment:
with all the data of migrations, which usually correlates with the plans. do we need both? can we use one to gather the data for both?

Reply:
@bkhizgiy by "migrations" you refer to VM migrations or is it the migration entity that corresponds to a plan like @liranr23 wrote above?

Reply:
its the migration entity that corresponds to plan, I left it because the original team had it, but do we really need both? I think one of them should be enough but which one, plans or migrations would be better?

Reply:
Following our conversation, moved all of the functionality to the migrations as well (like mode and destination), and added some additional metrics with coloration to the plan for additional information like time and total bytes transferred for extra calculations.